updater/internal/install/install.go

```diff
@@ -93,7 +85,7 @@ func (i Installer) Install(rb rollback.ActionAppender) error {
 
 // copyFiles moves the file tree rooted at latestDirPath to installDirPath,
 // skipping configuration files. Appends CopyFileAction-s to the Rollbacker as it copies file.
-func copyFiles(logger *zap.Logger, inputPath, outputPath, tmpDir string, rb rollback.ActionAppender) error {
+func copyFiles(logger *zap.Logger, inputPath, outputPath, installDir string, rb rollback.ActionAppender) error {
```

Comment:
This is only used in this file right? Doesn't seem necessary to have both outputPath and installDir since they ultimately will be the same thing?

Reply:
Good catch, refactored this.
I also gave it a better suited name that seems less general.